charts/seed-bootstrap/charts/etcd-druid/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcd-druid
+  namespace: garden
+  labels:
+{{ toYaml .Values.labels | indent 4 }}
+---
+apiVersion: {{ include "deploymentversion" . }}
+kind: Deployment
```

Comment:
Shouldn't we add VPA for the `etcd-druid`? Especially, because it is now a central component in the seed cluster?

Reply:
Done